ynr/apps/candidates/migrations/0050_delete_imageextra.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2018-09-20 15:41
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("candidates", "0049_cleanup_post_extra")]
+
+    operations = [
+        migrations.RemoveField(model_name="imageextra", name="base"),
+        migrations.RemoveField(model_name="imageextra", name="uploading_user"),
+        migrations.RemoveField(model_name="organizationextra", name="base"),
+        migrations.AlterField(
+            model_name="extrafield",
+            name="type",
```

Comment:
I'm confused about why this changes in this migration. Can you clarify?

Reply:
This was a mistake on my part because there are still python 3 related unicode migrations left to apply, and I didn't remove it.

I think I'll just do those migrations in a single PR after this, to save me having to pick them out all the time 🙄